poshi-runner/src/main/java/com/liferay/poshi/runner/elements/PoshiNodeFactory.java

```
@@ -143,6 +137,16 @@
 		return null;
 	}
 
+	private static DefinitionPoshiElement _getDefinitionPoshiElement() {
+		for (PoshiElement poshiElement : _poshiElements) {
+			if (poshiElement instanceof DefinitionPoshiElement) {
+				return (DefinitionPoshiElement)poshiElement;
+			}
+		}
+
+		return new DefinitionPoshiElement();
```

@kenjiheigel Should this new DefinitionPoshiElement be added to _poshiElements before returning it?

---

@pyoo47, I put that there as more of a safeguard. Thinking about it now though, maybe it should throw an error though. It should always have the static definition element in _poshiElements.